### PR TITLE
feat: Auto-approve Dependabot pull requests

### PR DESCRIPTION
**Pull request by l0b0**

### CODE REVIEW

**Review by blacha — CHANGES_REQUESTED** [review]
I do not think we should auto merge dependency changes.

the auto approve can be useful though.

**Comment by l0b0** [comment]
> I do not think we should auto merge dependency changes.

Including minor/patch ones? Basically, if we're not auto-merging something, then we're basically _requiring_ someone to do a more thorough check than the workflows in verifying that it's still working. Is that the case?

File: .kodiak.toml
Comment (paulfouquet) [inline .kodiak.toml:11]: As discussed, `major` is probably not a good idea for this repo

Comment (l0b0) [reply .kodiak.toml]: That's fair; I've changed it. I'm not sure what extra manual validation reviewers are meant to be doing of major updates though.

**Comment by l0b0** [comment]
Ping? Since approving these PRs is more than zero work per PR (plus the noise in my inbox and Slack), I'd love to see a solution which didn't involve a bunch of clicks for no reason.

**Comment by l0b0** [comment]
Changed to only auto-approve.

**Review by paulfouquet — APPROVED** [review]

**Comment by l0b0** [comment]
@blacha Could you please review this? We can't merge without your approval.

**Comment by l0b0** [comment]
Ping @blacha

**Comment by l0b0** [comment]
Ping @blacha

**Comment by blacha** [comment]
Kodiak is deprecated inside of LINZ and should be removed from repositories not expanded in use.

**Pull request closed without merging**
